components/mpas-ocean/cime_config/config_pes.xml

```diff
@@ -235,34 +235,34 @@
     <mach name="chrysalis">
       <pes compset="DATM.+MPASO" pesize="any">
         <comment>mpas-ocean+chrysalis: standard-res, compset=DATM+MPASO, 15x32x2 NODESxMPIxOMP</comment>
-        <MAX_MPITASKS_PER_NODE>32</MAX_MPITASKS_PER_NODE>
+        <MAX_MPITASKS_PER_NODE>64</MAX_MPITASKS_PER_NODE>
```

Comment:
@vanroekel, the changes in this file seem unrelated to this PR and may have been added inadvertently.  They may be a good idea but probably should be in another PR if so.